.travis/cover.py

```diff
@@ -23,7 +23,7 @@
 
 def main():
     # Run tests with virtual frame buffer for X support.
-    if call('xvfb-run -a cylc test-battery --chunk $CHUNK --state=save -j 5',
+    if call('xvfb-run -a cylc test-battery --chunk $CHUNK --state=save -j 5 tests/authentication tests/cylc-scan/03-monitor.t',
```

Comment:
Need to revert this change

Reply:
@oliver-sanders, no probs, I'll sort that out now.